dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -221,7 +221,7 @@ public AbstractInterfaceConfig(ModuleModel moduleModel) {
     /**
      * The url of the reference service
      */
-    protected final transient List<URL> urls = new ArrayList<URL>();
+    protected final transient List<URL> urls = new ArrayList<>();
```

Comment:
**Security Issue**: Usage of raw types in generic collections can lead to ClassCastException if the content of the collection is not what is expected, breaking type safety. <br> **Fix**: Replace raw type usage with parameterized type in the collection to ensure type safety. <br> **Code Suggestion**: 
 ```
 -    protected final transient List<URL> urls = new ArrayList<URL>();
 +    protected final transient List<URL> urls = new ArrayList<>();
 ```